lib/access_import/acc_utils.rb

```diff
@@ -81,7 +81,7 @@ def language_skills_attributes(sprachen)
     sprachen.map do |sprache|
       [
         (Time.now.to_f * 1000).to_i,
-        { language: sprache[:language][:lang], level: sprache[:kenntnisstufe_ve] }
+        { language: sprache[:language][:lang], level: sprache[:kenntnisstufe_ve] || 'basic' }
```

Comment:
If `sprache[:kenntnisstufe_ve]` can also be an empty string (rather than `nil`) you can use `sprache[:kenntnisstufe_ve].presence || 'basic'`